t/plugin/kafka-logger.t

```diff
@@ -722,3 +722,43 @@ GET /t
 [qr/partition_id: 1/,
 qr/partition_id: 0/,
 qr/partition_id: 2/]
+
+
+
+=== TEST 20: add plugin metadata
```

Comment:
Should add another test case to cover whether the `log_format` setting is effective.